src/main/webapp/app/modules/user-profile/complete-user-profile.tsx

```diff
@@ -59,6 +60,7 @@ class CompleteUserProfile extends React.Component<ICompleteUserProfileProps, {}>
               <p>Loading...</p>
             ) : (
               <AvForm model={userProfile} onSubmit={this.completeProfile}>
+                {window.console.log(userProfile)}
```

Comment:
this one is for debugging?

Reply:
@HowardYaw I once noticed you wrote `window.console.log` for other pages as well. Is it intentional to log it to the console?

Reply:
@kachun333 which page?

Reply:
@CarlosSia removed on 9022f20616347651d8f9daf441652f7f22ec6849